bin/a0-ext

```diff
@@ -147,6 +149,28 @@ prog
           return process.exit(0);
         });
       });
+  })
+  .command('deploy', 'Package assets for extension deployment')
+  .option('--token', 'Access token')
+  .option('--package', 'Path to extension bundle')
+  .option('--url', 'URL to extension deploy endpoint')
+  .action((args, opts) => {
+    const rootPath = process.cwd();
+    const filePath = path.join(rootPath, opts.package || 'package.json');
```

Comment:
Why is it defaulting to `package.json`? This command should take the output of the `build`  command, which should be a zip file.

Reply:
Right. It should be `dist/package.zip` by default. Fixed.